selenium/exercise_6.rb

```diff
@@ -0,0 +1,30 @@
+require 'selenium-webdriver'
+class ClickElelemt
+    attr_accessor :driver, :wait
+
+    def initialize(driver_path)
+        Selenium::WebDriver::Chrome.driver_path=driver_path
+        @wait = Selenium::WebDriver::Wait.new(:timeout => 10)
+        @driver=Selenium::WebDriver.for :chrome
+        driver.manage.timeouts.implicit_wait =10
+    end
+    def open_browser(url)
+        driver.get(url)
+    end
+    def search_informatio
+        dropdown_list = wait.until{driver.find_element(:css ,"body > app-root:nth-child(1) > app-forms:nth-child(3) > section:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > form:nth-child(1) > div:nth-child(2) > div:nth-child(2) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > select:nth-child(1)")}
```

Comment:
A better way to interact with dropdowns would be using the Selenium::WebDriver::Support::Select module
dropdown = Selenium::WebDriver::Support::Select.new(dropdown_element)

Refer to documentation for more details.